### DISCUSSION TITLE
react-storefront stable branch

### DISCUSSION THREAD

**Discussion by AbduRahmanS** [saleor/saleor#10169] [Q&A] [answered]
Hello, 
I'm building a small e-commerce using saleor and I want to use the react-storefront repository for the storefront, i didn't find any releases and i'm not sure which branch to use. I guess the main branch is the most stable but i need to confirm that before start making changes.
So which branch can i use to build my storefront
Thanks, in advance,

PS: I'm learning English, sorry for grammatical or spelling error

**Comment by karolkielecki** [comment, marked as answer]
Hi @AbduRahmanS! Thanks for asking. The react-storefront is a starter kit for now. It still misses some features and needs improvements, etc. So, it's not a ready-to-use project yet. However, if you want to use it, the main branch is the one you want. Let us know if you have more questions.

Cheers, and good luck!
